apps/studio/components/interfaces/Support/SupportFormV2.tsx

```diff
@@ -0,0 +1,696 @@
+              </FormItemLayout>
+            )}
+          />
+          {organizationSlug &&
+            subscriptionPlanId !== 'enterprise' &&
+            category !== 'Login_issues' && (
+              <PlanExpectationInfoBox
+                orgSlug={organizationSlug}
+                projectRef={projectRef}
+                planId={subscriptionPlanId}
+              />
+            )}
+        </div>
+
+        <div className="px-6 grid sm:grid-cols-2 sm:grid-rows-1 gap-4 grid-cols-1 grid-rows-2">
+          <FormField_Shadcn_
+            name="category"
+            control={form.control}
+            render={({ field }) => (
+              <FormItemLayout layout="vertical" label="What areas are you having problems with?">
+                <FormControl_Shadcn_>
+                  <Select_Shadcn_
+                    {...field}
+                    defaultValue={field.value}
+                    onValueChange={field.onChange}
+                  >
+                    <SelectTrigger_Shadcn_ className="w-full">
+                      <SelectValue_Shadcn_>
+                        {CATEGORY_OPTIONS.find((o) => o.value === field.value)?.label}
+                      </SelectValue_Shadcn_>
+                    </SelectTrigger_Shadcn_>
+                    <SelectContent_Shadcn_>
+                      <SelectGroup_Shadcn_>
+                        {CATEGORY_OPTIONS.map((option) => (
+                          <SelectItem_Shadcn_ value={option.value}>
+                            {option.label}
+                            <span className="block text-xs opacity-50">{option.description}</span>
+                          </SelectItem_Shadcn_>
+                        ))}
+                      </SelectGroup_Shadcn_>
+                    </SelectContent_Shadcn_>
+                  </Select_Shadcn_>
+                </FormControl_Shadcn_>
+              </FormItemLayout>
+            )}
+          />
+          <FormField_Shadcn_
+            name="severity"
+            control={form.control}
+            render={({ field }) => (
+              <FormItemLayout layout="vertical" label="Severity">
+                <FormControl_Shadcn_>
+                  <Select_Shadcn_
+                    {...field}
+                    defaultValue={field.value}
+                    onValueChange={field.onChange}
+                  >
+                    <SelectTrigger_Shadcn_ className="w-full">
+                      <SelectValue_Shadcn_ placeholder="Select a project">
+                        {field.value}
+                      </SelectValue_Shadcn_>
+                    </SelectTrigger_Shadcn_>
+                    <SelectContent_Shadcn_>
+                      <SelectGroup_Shadcn_>
+                        {SEVERITY_OPTIONS.map((option) => (
+                          <SelectItem_Shadcn_ value={option.value}>
+                            {option.label}
+                            <span className="block text-xs opacity-50">{option.description}</span>
```

Comment:
don't use opacity-50

Reply:
text-foreground-light, text-foreground-lighter

Reply:
ahh yup! good catch - sorry this was old code